src/test_workflow/compare_benchmark/compare_args.py

```diff
@@ -0,0 +1,41 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+
+
+class CompareArgs:
+    id1: str
+    id2: str
+    results_format: str
+    results_numbers_align: str
+    results_file: str
+    show_in_results: str
+
+    def __init__(self) -> None:
+        self.parser = argparse.ArgumentParser(description="Compare two IDs")
+        self.parser.add_argument("id1", type=str, help="The baseline ID to compare")
+        self.parser.add_argument("id2", type=str, help="The contender ID to compare")
+        self.parser.add_argument("--results-format", type=str, help="Defines the output format for the results, markdown or csv (default: markdown)")
+        self.parser.add_argument("--results-numbers-align", type=str, help="Defines the format for the command line results. (Default: right)")
+        self.parser.add_argument("--results-file", type=str, help="File path to write the results file to")
+        self.parser.add_argument("--show-in-results", type=str, help="Determines whether to include the comparison in the results file")
```

Comment:
Can you add (Defaults) to all these optional parameters?
Also use Capital for all of them.

Reply:
Te defaults will be set by OSB itself when these parameters are not passed. 
i'm okay with anything here.